### DISCUSSION TITLE
是否考虑支持更多的平台的api？

### DISCUSSION THREAD

**Discussion by GinWU05** [BurnHair/docs#126] [功能建议]
现在除了OpenAI，会考虑接入Claude的api吗？

感谢BennyThink提供的平台和认真维护。

**Comment by BennyThink** [comment]
😂没啊，等以后的，现在 OpenAI的都够我喝一壶了

**Comment by BennyThink** [comment]
https://github.com/orgs/BurnHair/discussions/207

现在有了，以后准备加入 Mistral AI
